### ISSUE TITLE
D_Loss and G_loss gives Nan value after 10 epochs on horse2zebra dataset.

### ISSUE THREAD

**Issue by israrbacha** [taki0112/DRIT-Tensorflow#9]
i run DRIT code on horse2zebra dataset after 10 epochs it gives nan for g_loss and d_loss i use original gan loss function.

**Comment by ZzzTD** [comment]
May I ask how you solved this problem？
